sql/core/src/main/scala/org/apache/spark/sql/api/r/SQLUtils.scala

```diff
@@ -155,6 +155,52 @@ private[sql] object SQLUtils {
     df.mapPartitionsInR(func, packageNames, bv, realSchema)
   }
 
+ /**
+  * The helper function for gapply() on R side.
+  */
+ def gapply(
+    df: DataFrame,
+    func: Array[Byte],
+    packageNames: Array[Byte],
+    broadcastVars: Array[Object],
+    schema: StructType,
+    col: String): DataFrame = {
+
+   val realSchema =
+      if (schema == null) {
+        SERIALIZED_R_DATA_SCHEMA
+      } else {
+        schema
+      }
+
+   val dfSchema = df.select(df(col)).schema
+   if (dfSchema.length == 0) throw new IllegalArgumentException(s"Invaid column name $col")
+   val dataType = dfSchema(0).dataType
+
+   val sqlContext = df.sqlContext
```

Comment:
This is unusual:

The following fails: import df.sqlContext.implicits._

But this works: 
val sqlContext = df.sqlContext
import sqlContext.implicits._

I'm seeing this after pulling the latest changes. Any ideas ?